docs/imaging_model/README.md

```diff
@@ -0,0 +1,64 @@
+# Imaging and Report Model
+
+## Motivation
+
+Participants often have files in addition to genomic files ranging from
+MRI images, histology images, and reports.
+Some of these file types relate directly to the biospecimen rather than the
+participant, eg: proteomics and histology.
+In order to capture and distribute these files, we need to model them
+appropriately.
+
+## Implementation
+
+Files will need to be distributed through gen3, so the models will need to
+inherit the Indexd model similar to GenomicFiles.
+Each new file type will have a unique prefix for the model as well as specific
+metadata fields unique to that type of file.
+
+![Imaging Model](ImagingModelTables.png)
+
+### Radiology File and Report
+
+RadiologyFile and RadiologyReport have a many-to-many relationship to
+each-other and one-to-many from Participant.
+When linking a file with a report, we must ensure that they both share the
+same participant. This is similar to what we do when linking biospecimens and
+diagnoses.
+
```

Comment:
Also realizing we forgot to include operations reports (they are listed in the original ticket). These are actually the surgical operations reports and this is starting to highlight how we're really missing the concept of a clinical event to relate the appropriate images, reports, biospecimens and diagnoses. It may be reasonable to associate them directly with the participant, but in reality they're tangentially associated with certain radiology images/reports (pre/post op), but wouldn't necessarily be related to follow up radiology images/reports. The problem here is the same that caused us to associate diagnoses and biospecimens. If I'm looking at a genomic file, how do I know which radiology/operation data is associated with that genomic data, as a patient can have multiple different tumors over time.